src/getServerOptions.ts

```diff
@@ -4,7 +4,7 @@ import { JavaConfig } from "./getJavaConfig";
 interface GetServerOptions {
   metalsClasspath: string;
   serverProperties: string[] | undefined;
-  clientName: string;
+  clientName?: string;
```

Comment:
why not removing this entirely? (Maybe I’m missing some context)

Reply:
Yea so I had it this way at first, but then if this isn't set and someone upgrades the client extension, but wants to use an old Metals version, then everything won't be set as expected. So I didn't fully know what level of backwards compatibility we wanted from client to server. What do you think?

Reply:
Replied in the other thread (which I had initially missed)